docs/using.md

```diff
@@ -171,41 +489,294 @@ is passed in `infrastructure_provisioning: bar` and the third `arbitrary_key: ba
 
-```yaml
-AutoRun:
-  - When:
-      mongodb_setup:
-        $eq: standalone
-```
+    AutoRun:
+      - When:
+          mongodb_setup:
+    	$eq: standalone
 
 Let's say this is `DemoWorkload` again. In this case, if `mongodb_setup` is `standalone`
 we schedule `demo_workload` with no additional params.
 
 A few notes on the syntax:
-- Supports multiple When/ThenRun blocks per AutoRun. Each are evaluated independently.
-- When blocks can evaluate multiple conditions. All conditions must be true to schedule the task.
-- When supports $eq and $neq. Both can accept either a scalar or list of values.
-- For a list of values, $eq evaluates to true if it is equal to at least one.
-- For a list of values, $neq evaluates to true if it is equal to none of the values.
-- ThenRun blocks are optional.
-    - **Most usecases do not need to use ThenRun**
-    - If you do use ThenRun, please be judicious. If you have a task that is scheduled when
-      mongodb_setup == replica, it would be confusing if mongodb_setup was overwritten to standalone.
-      But it would be ok to overwrite mongodb_setup to replica-delay-mixed, as is done in the
-      [ParallelWorkload][pi] workload.
-- Each item in the ThenRun list can only support one {bootstrap_key: bootstrap_value} pair.
-- If using ThenRun but you would also like to schedule a task without any bootstrap overrides,
-  Add an extra pair to ThenRun with the original key/value, like done on line 189 [here][pi].
-- If using ThenRun, the new task name becomes <taskname>_<bootstrap-value>. In the ParallelWorkload example,
-  the task name becomes parallel_insert_replica_delay_mixed (name is automatically converted to snake_case).
-  The bootstrap-key is not included in the name for the purpose of not changing existing names and
-  thus deleting history. This may change after PM-2310.
-
-NB on patch-testing:
-
-1.  After the task runs you can call `set-module` again with more local changes from Genny or DSI.
-    This lets you skip the 20 minute wait to recompile the server.
-2.  Alternatively, you can follow the instructions at the top of the `system_perf.yml`
-    file for information on how to skip compile.
-3.  You can restart workloads from the Evergreen Web UI.
+
+-   Supports multiple When/ThenRun blocks per AutoRun. Each are evaluated independently.
+-   When blocks can evaluate multiple conditions. All conditions must be true to schedule the task.
+-   When supports $eq and $neq. Both can accept either a scalar or list of values.
+-   For a list of values, $eq evaluates to true if it is equal to at least one.
+-   For a list of values, $neq evaluates to true if it is equal to none of the values.
+-   ThenRun blocks are optional.
+    -   ****Most usecases do not need to use ThenRun****
+    -   If you do use ThenRun, please be judicious. If you have a task that is scheduled when
+        mongodb<sub>setup</sub> == replica, it would be confusing if mongodb<sub>setup</sub> was overwritten to standalone.
```

Comment:
`<sub>`
(I think ensure there's no html in this doc except for the `<a>`s for headings/sections which are....actually not explicitly needed since github renders `<a name="the-title"><h1>The Title</h1></a>` or something similar...